website/components/smart-link/index.jsx

```diff
@@ -0,0 +1,41 @@
+import Link from 'next/link'
+
+export default function SmartLink({
+  href,
+  as,
+  replace,
+  scroll,
+  shallow,
+  passHref,
+  children,
+  external,
+  ...props
+}) {
+  const isAbsoluteUrl = new RegExp('^(?:[a-z]+:)?//', 'i')
+
+  if (!isAbsoluteUrl.test(href)) {
+    return (
+      <Link
+        href={href}
+        as={as}
```

Comment:
I think `next@10` has done away with the `as` prop - we _just_ upgraded. Double check me here as well!

Reply:
Ahh yeah, seems it is still available but is in fact deprecated.  This was an artifact from utilizing other `<SmartLink />` code so might be good to adjust it there as well!